logger/main.go

```diff
@@ -76,7 +76,7 @@ func main() {
 			if resp != nil && resp.Node != nil {
 				drainChan <- resp.Node.Value
 			}
-			time.Sleep(time.Duration(publishInterval))
+			time.Sleep(time.Duration(publishInterval) * time.Seconds)
```

Comment:
I couldn't get the `go` compiler to pass that version. [`time.Sleep`](http://golang.org/pkg/time/#example_Sleep) expects a `Duration`, and this is the [canonical way](http://golang.org/pkg/time/#Duration) to convert an integer to seconds.